models/IncomingMessage.cs

```diff
@@ -0,0 +1,23 @@
+using Newtonsoft.Json.Linq;
+
+namespace NosAyudamos
+{
+    public class IncomingMessage
```

Comment:
Is this really an *incoming*? I feel like both incoming and outgoing messages are going to have the same three properties at least.